student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(age, classroom, _name = 'Unknown', _parent_permission: true)
+    super(age, _name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] For this inheritance to work, you must import the Person class.

  ```
  require_relative './person'
  ```

- [ ] Initializing `parent_permission: true` on this line 3 is somewhat incorrect, the student can have `false`, but your code initializes `parent_permission` to `true` regardless of the input. Kindly update your `super` parameters here:

  ```
  super(age, name = 'Unknown', parent_permission: parent_permission)
  ```